### ISSUE TITLE
Minimum Sdk for Android is 19

### ISSUE THREAD

**Issue by lutakyn** [Tealium/tealium-react-native#47]
Why is the minimum sdk for Android 19 because my project minimum sdk is 16. Is there a reason for that.

This is the sample build.gradle file for tealium-react-native installed on my app

```
apply plugin: 'com.android.library'

//allprojects {
//    repositories {
//        maven {
//            url "http://maven.tealiumiq.com/android/releases"
//        }
//    }
//}

buildscript {
    repositories{
        jcenter()
        maven {
            url 'http://maven.tealiumiq.com/android/releases/'
        }
    }

    dependencies {
        classpath 'com.android.tools.build:gradle:2.3.3'
    }
}

android {
    compileSdkVersion 28
    buildToolsVersion "28.0.1"
    defaultConfig {
        minSdkVersion 19
        targetSdkVersion 28
    }
    buildTypes {
        release {
            minifyEnabled false
            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
        }
    }
}

dependencies {
    implementation 'com.facebook.react:react-native:+'
    implementation 'com.tealium:library:5.5.5'
    implementation 'com.tealium:lifecycle:1.1.2'
}
```

**Comment by craigrouse** [comment]
@lutakyn we only support Android API level 19+. This is because that's the minimum we support in the core Android library bundled with our React Native plugin. See here for more details: https://docs.tealium.com/platforms/android-java/install/

API 19 (KitKat) was released nearly 7 years ago, and there are very few devices using older versions of Android. We took our cues from Google's official stats, and KitKat is the oldest version of Android that still has significant user numbers: https://developer.android.com/about/dashboards

I hope this helps to explain. Let us know if you have any further questions. If supporting older Android versions is essential for you, it may be an option for you to fork the repo and change the Tealium library version to [5.5.2](https://github.com/Tealium/tealium-android/releases/tag/5.5.2), which was the version before we updated to API 19. This version supports API 9+. There have been some API changes since then, so it may be necessary to amend some of the React wrapper code in the plugin, and you won't benefit from any recent bugfixes or new features.
